client/src/components/RelatedItemsAndOutfit/RelatedItemsAndOutfit.jsx

```diff
@@ -1,18 +1,64 @@
 import React from 'react';
+import server from './Axios';
+import RelatedProducts from './RelatedProducts/RelatedProducts';
+import UserOutfit from './UserOutfits/UserOutfit';
 
 class RelatedItemsAndOutfit extends React.Component {
   constructor(props) {
     super(props);
-
+    this.server = server;
```

Comment:
Do you absolutely need to attach server to this component? The server is an imported axios instance, does it work without 'this' on line 24 minus this declaration on line 9?

Reply:
Yeah so I had it like that originally, but if I don't attach it to the class itself, esLint AirBnB gets angry and says 'must use 'this' in methods of this class', so I had to attach it to the constructor

Reply:
I couldn't replicate it, very curious! But the error was  "warning  Expected 'this' to be used by class method". I changed it to not use 'this'.